pkg/harvester/config/doc-links.js

```diff
@@ -0,0 +1,2 @@
+
+export const DOC_LINKS = { CONSOLE_URL: 'https://docs.harvesterhci.io/v1.4/host/' };
```

Comment:
Hi @WebberHuang1118 , we can update this link once we have a document block describe how to setup console URL.

Reply:
Is there a way not to hard-code the version? (and determine from current running branches).

Reply:
Sounds great, as long as the version in the package.json file is bumped correctly on each place.